common/src/main/java/net/william278/huskhomes/config/Settings.java

```diff
@@ -215,6 +215,10 @@ public Optional<String> getSoundEffect(@NotNull SoundEffectAction action) {
     @YamlKey("economy.enabled")
     public boolean economy = false;
 
+    @YamlComment("Use this currency for payments (works only with RedisEconomy), defaults to Vault currency")
+    @YamlKey("economy.currency")
```

Comment:
`economy.redis_economy_name` perhaps? And then get rid of the comment, we can add docs for this and don't want _too_ many comments filling up the file if possible.
Also, style ;-)

Reply:
Done. replaced `economy.currency` key with `economy.currency_name`